### PR TITLE
doc(customize): Added paragraph about adding new elements

### PR DESCRIPTION
**Pull request by edouardmenayde**

### CODE REVIEW

**Comment by edouardmenayde** [comment]
@bas080 what do you think we could add here ?

File: doc/customize.md [hunk 1]
```diff
@@ -100,3 +100,120 @@ export class SomeForm extends Form {
...
+
+## Adding an element
+
+Not only can you can customize existing form elements, but you can use the
+power of `aurelia-view-manager` to add new elements.
+
+> main.js
+
+```js
+
```
Comment (RWOverdijk) [inline doc/customize.md:112]: Remove extra newlines

File: doc/customize.md [hunk 2]
```diff
@@ -100,3 +100,120 @@ export class SomeForm extends Form {
...
+```
+
+Besides registering the `image-select` component we also need to create it in
+the '/elements' directory.
+
+> /elements/image-select.js
+
+```js
+
+
```
Comment (RWOverdijk) [inline doc/customize.md]: Remove extra newlines

File: doc/customize.md [hunk 3]
```diff
@@ -100,3 +100,120 @@ export class SomeForm extends Form {
...
+    key:  'picture',
+    type  'image-select'
+  }];
+
+  this.userSettingsModel = {/* ... */};
+
+```
+
+> someViewModelView.html
+```
```
Comment (RWOverdijk) [inline doc/customize.md]: Add ```html

Comment (RWOverdijk) [inline doc/customize.md]: Newline

**Pull request merged**